e2e/plugins/index.ts

```diff
@@ -0,0 +1,3 @@
+// Plugins enable you to tap into, modify, or extend the internal behavior of Cypress
+// For more info, visit https://on.cypress.io/plugins-api
+module.exports = (on, config) => {};
```

Comment:
This logs an error in my WebStorm IDEA because node is somehow not configured in the `tsconfig.json`. Do you have any idea how this can be fixed?